CODEOWNERS

```diff
@@ -110,6 +110,7 @@
 /Workbooks/ConnectedVehicle/** @microsoft/mcvp-workbooks
 
 /Workbooks/Azure*Update*Center/** @microsoft/azure-update-center
+/Workbooks/Azure*Monitor*SCOM*managed*instance/** @microsoft/azure-monitor-scommi
```

Comment:
is there a reason you are calling everything "azure monitor scom managed instance" ?  is there any reason this isn't literally just "SCOM managed instance" without the "Azure monitor" ?  or even shorter, the folder could be "SCOM-MI" 

the other things with azure monitor in the name have that in the name because it is literally the name of that resource type or are an azure monitor feature, like "Azure Monitor Workspace" (which is an actual literal resource type)